packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -997,6 +1010,10 @@ export class StatsAnalyzer extends EventsScope {
       this.statsResults[mediaType][sendrecvType].headerBytesSent = result.headerBytesSent;
       this.statsResults[mediaType][sendrecvType].retransmittedBytesSent =
         result.retransmittedBytesSent;
+      this.statsResults[mediaType][sendrecvType].isRequested =
+        result.isRequested ||
+        (result.lastRequestedUpdateTimestamp &&
+          performance.timeOrigin + performance.now() - result.lastRequestedUpdateTimestamp < 60000);
```

Comment:
I suppose here is also need to be `MQA_INTERVAL`

Reply:
Done